generate.py

```diff
@@ -156,4 +157,8 @@ def main(
     from jsonargparse import CLI
 
     torch.set_float32_matmul_precision("high")
+    warnings.filterwarnings(
```

Comment:
Can you add a comment that mentions which component is triggering this warning?

Reply:
Added. It comes from the empty tensor (through our use of EmptyInitOnDevice)